feed/templates/base.html

```diff
@@ -58,7 +58,7 @@
 
 
 
-{% block sidebar %} <!--- side bar content--> {% endblock %}
```

Comment:
Deleting your sidebar? Was this on purpose? I just want to double check.

Reply:
yes because it is injected from it's own page right into the index.html
it was easier to control that way      {% block sidebar %} {% include 'sidebar.html' %} {% endblock sidebar %}